src/object/player.hpp

```diff
@@ -26,6 +26,7 @@
 #include "supertux/player_status.hpp"
 #include "supertux/sequence.hpp"
 #include "supertux/timer.hpp"
+#include "trigger/climbable.hpp"
 #include "video/surface_ptr.hpp"
```

Comment:
No need to include a file in the header if it's only used in the body; include in the body instead. If we include too much in the headers, we risk creating a dependency loop

Reply:
I thought it would be in the header since there is a `Climbable&` definition for `m_climbing`. I'm fixing that right now

Reply:
If it wasn't there then there is no need to add it. Most of the time, when you need to add a certain class, you can do just `Class NameOfTheClass;` in the header, since you don't actually use the class in the header, and you just include in the body.